app/League-Entries/page.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { LeagueEntries } from '@/components/LeagueEntries/LeagueEntries';
+import { LeagueSurvivors } from '@/components/LeagueSurvivors/LeagueSurvivors';
+import * as React from 'react';
+
+/**
+ * @returns Leagues
+ */
+const Leagues: React.FC = (): React.JSX.Element => {
```

Comment:
It's not required to type the component. 
```suggestion
const Leagues = (): JSX.Element => {
```

Reply:
Updated in latest commit.